README.md

```diff
@@ -46,6 +46,26 @@ except Exception:
     logger.exception("Bad math.")
 ```
 
+###setup for Airbrake alternatives (e.g. Errbit)
```

Comment:
@joegod - perhaps the wording `Airbrake alternatives` needs to be changed. Maybe

- Airbrake compatible backends
- Airbrake compliant backends

Just a thought.